### PR TITLE
[#4] Allow removal of a task

### LINKED ISSUE #4: Add support for state=absent
Otherwise there is no way (other than manually) to remove a task.

Especially handy when renaming a supervisor task

### PR DESCRIPTION
**Pull request by sujaymansingh**
Closes #4

### CODE REVIEW

File: tasks/main.yml [hunk 1]
```diff
@@ -11,6 +11,12 @@
 - name: Installing task {{ name }}
   template: src=task.conf.j2 dest={{ supervisor_config_dir }}/{{ name }}.conf
   sudo: yes
+  when: state == "present"
```
Comment (bobbyrenwick) [inline tasks/main.yml]: I think these should be wrapped in curly braces to make Ansible 2 happy `"{{ state == 'present' }}"`

File: tasks/main.yml [hunk 2]
```diff
@@ -11,6 +11,12 @@
...
+
+- name: Removing task {{ name }}
+  file: path={{ supervisor_config_dir }}/{{ name }}.conf state=absent
+  sudo: yes
+  when: state == "absent"
```
Comment (bobbyrenwick) [inline tasks/main.yml]: As above

**Comment by bobbyrenwick** [comment]
A couple of small comments but otherwise LGTM

**Comment by pkatseas** [comment]
LGTM after the already pointed out comments

**Comment by sujaymansingh** [comment]
@bobbyrenwick @pkatseas Are you happy with me merging this?

**Comment by pkatseas** [comment]
Yep

**Pull request merged**